R/mod_quality_overview.R

```diff
@@ -100,8 +100,7 @@ mod_quality_overview_server <- function(input,
       register_data[["hospital_name_structure"]] %>%
           dplyr::select(
             .data[[config$data$column$unit_name$hfshort]],
-            .data[[config$data$column$unit_id$hf]]
-
+            .data[[config$data$column$unit_id$hf]],
```

Comment:
OK with comma here?

Reply:
did not notice that one. I will remove it.  you are getting very good at the reviewing:-)

Reply:
Close to perfect:)